parm/soca/soca_fix_stage_025.yaml.j2

```diff
@@ -1,4 +1,3 @@
-# TODO(AFE): make resolution dependent
```

Comment:
Why are these not made resolution dependent? Seems like a lot of redundancy here that can be simplified with some Jinja

Reply:
This got broken out into different yamls - @guillaumevernieres is there a particular approach you're committed to?

Reply:
Yes, the one that we have now. We might decide to combine these at some point, but it's pretty far down the priority list.